lib/oli_web/templates/email/account_already_exists.html.eex

```diff
@@ -1,12 +1,23 @@
-<div class="flex flex-column gap-y-4">
-  <h3>Are you trying to create a new account?</h3>
-  <p>
-    Someone tried to create an account with this email but an account already exists.
-    If this was you, you can login <%= link "here", to: @url, target: "_blank" %> with your existing email and password.
-    If you forgot your password, you can reset it by clicking <%= link "Forgot Password?", to: @forgot_password, target: "_blank" %>.
-  </p>
-  <p>If this was not you, you can disregard this email.</p>
-</div>
-
-
-
+<table align="center" role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%" style="margin: auto;">
```

Comment:
Email clients typically don't render html in the same sane way that browsers do, further they certainly do not have access to tailwind classes. I updated this template to be email friendly like the other templates and better match the typical email style since this in particular is an email we want a user to see as authentic